datahub-actions/src/datahub_actions/plugin/source/kafka/kafka_event_source.py

```diff
@@ -86,11 +86,13 @@ def kafka_messages_observer(labels: Dict[str, str] = {}) -> Callable:
         name="kafka_offset",
         documentation="Kafka offsets per topic, partition",
         labelnames=["topic", "partition", *labels.keys()],
+        namespace=labels.get('pipeline_name')
```

Comment:
hey, cloud you call this label `pipeline_name`, asking this because `namespace` will be overwritten by Prometheus when it scrapes, this metric.

Reply:
Hmm I'm not sure how Prometheus configurations work but I've tried setting it like 
```
labelnames=["topic", "partition", *labels.keys(), labels.get('pipeline_name')]
```
and the error still persists.